pyproject.toml

```diff
@@ -10,6 +10,7 @@ dependencies = ["astropy"]
 dynamic = ["version"]
 
 [project.optional-dependencies]
+viz = ["qgridnext", "lineid_plot"]
 test = ["pytest-astropy"]
```

Comment:
These other optional deps probably aren't needed any more, right?

Reply:
nope, I should remove it